public/javascripts/settingsController.js

```diff
@@ -14,6 +14,14 @@ var settingsController = function($scope, $http, $location) {
     });
   };
 
+  $scope.delUser = function(username, userArray, delUserIdx) {
+    $http.post('api/delUser/'+username._id).then(function successCallback(response) {
+      userArray.splice(delUserIdx, 1);
```

Comment:
It seems like a better solution is to have the route return all the users in the `response`, and then to update the users to the list of users in the response.